### ISSUE TITLE
Add support for global ignore directories

### ISSUE THREAD

**Issue by mjpitz** [depscloud/depscloud#137]
Right now, when we want to add a common directory to ignore, we need to go to each integration and add an item. Adding support for some common global ignores would be really useful. Some common directories to ignore are often:

```
vendor/
test/
testdata/
hack/
```

**Comment by PankhudiB** [comment]
You mean to be ignored by git checkin ?

**Comment by mjpitz** [comment]
@PankhudiB no. The extractor process matches files based on a set of patterns. In this [commit](https://github.com/depscloud/depscloud/commit/d84e2b98433cc8e09bae232deffa7cbfe417b429) I went through and added `testdata` as an ignore for every extractor. This can be tedious. Having a way to match across extractor patterns would be much more useful rather than needing to update each and every one of them.
